django/tasks/signals.py

```diff
@@ -0,0 +1,4 @@
+from django.dispatch import Signal
+
+task_enqueued = Signal()
+task_finished = Signal()
```

Comment:
Hmmm... in background systems I've used I've often added some code at the beginning of tasks (in particular to do some observations on the gap between when a task is enqueued and when it is started properly, or to add some basic instrumentation to have an idea of in-flight task behavior), but to be honest I'm able to do that with a custom task wrapper.

I think it might still be valuable for observability tooling to ship Django plugins or the like, but we don't need it immediately.